docs/bio2.md

```diff
@@ -0,0 +1,5 @@
+asdk jah dakjhsd kajshd 
+zx czxc zc zxc 
+
+
+Go to the "Files changed" tab and add a line comment to the Pull Request mentioning that you should add your favorite color
```

Comment:
Go to the "Files changed" tab and add a line comment to the Pull Request mentioning that you should add your favorite color